core/src/main/java/com/novoda/noplayer/NoPlayer.java

```diff
@@ -311,12 +312,16 @@
 
     interface AdvertListener {
 
+        void onAdvertsLoadError(Exception cause);
```

Comment:
Does this notify about the same exception we pass in to the `AdvertsLoader.Callback`?

Reply:
yes is the answer, I just needed to scroll down 😂